ui/src/app/store/zone/constants.ts

```diff
@@ -0,0 +1,26 @@
+/* The following constants should be made generic, usable across models. */
+export const ACTION_STATUS = {
+  failed: "failed",
+  processing: "processing",
+  successful: "successful",
+} as const;
+/***************************/
+
+export const ZONE_PK = "id";
+
+export const ZONE_MODEL = "zone";
+
+export const ZONE_ACTIONS = {
+  cleanup: "cleanup",
+  create: "create",
+  delete: "delete",
+  fetch: "fetch",
+  update: "update",
+} as const;
+
+export const ZONE_WEBSOCKET_METHODS = {
+  create: "create",
+  delete: "delete",
+  list: "list",
+  update: "update",
+} as const;
```

Comment:
We typically only use these strings once ever, but I think housing it in a `ZONE_WEBSOCKET_METHODS` const object makes it clear when we're referring to an "action" (as we refer to it in the UI) and when we're referring to a "websocket method" (as expected by the API)